### PR TITLE
Update for main branch rename

### PR DESCRIPTION
**Pull request by syg**

### CODE REVIEW

**Comment by syg** [comment]
@rwaldron Not sure who owns this repo, can you merge this?

**Comment by rwaldron** [comment]
@syg I don't have access to this repo 🤷🏼‍♂️

**Comment by syg** [comment]
Hm, @leobalter do you know who owns this repo?

**Review by J-Vin — APPROVED** [review]

**Review by J-Vin — APPROVED** [review]

**Comment by leobalter** [comment]
@syg I'm sorry I missed the notification earlier. I believe @jugglinmike might know the answer to this.

**Comment by jugglinmike** [comment]
Hi @syg! Hi @leobalter!

I can merge this, but maybe we can improve it a bit, first. What do you think about tolerating either version of the license? That would alleviate consumers from the need to synchronize versions of Test262 and this tool.

That's admittedly a bit on the conservative side, but understanding how this software is used was hard enough even when we first released it. Following years of inactivity, I'm even more reluctant to change things. In this case (because we're already working in terms of a regular expression) the effort seems minimal. Hence my request :)

File: src/parseTestRecord.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
        r'// Use of this source code is governed by a BSD-style license that can be[\r\n]{1,2}' +
        r'// found in the LICENSE file\.' +
        r'|' +
-       r'// See LICENSE or https://github\.com/tc39/test262/blob/master/LICENSE' +
+       r'// See LICENSE or https://github\.com/tc39/test262/blob/HEAD/LICENSE' +
```
Comment (leobalter) [inline src/parseTestRecord.py:38]:
```suggestion
       r'// See LICENSE or https://github\.com/tc39/test262/blob/(master|HEAD)/LICENSE' +
```

This won't mess up the matching group, I believe so.

I've created a very quick test here: https://gist.github.com/leobalter/ccee8b1d9dda74cfa4a1d21664a5bbdd

Out of curiosity, this regexp part exists to capture only two test files in Test262:

```
test/built-ins/Array/prototype/concat/S15.4.4.4_A3_T2.js
test/built-ins/Array/prototype/concat/S15.4.4.4_A3_T3.js 
```

**Comment by jugglinmike** [comment]
Thanks, @leobalter! Thanks, @syg!

**Pull request merged**